server/src/main/java/org/elasticsearch/cluster/routing/allocation/AllocationService.java

```diff
@@ -137,14 +138,28 @@ private ClusterState buildResult(ClusterState oldState, RoutingAllocation alloca
             .routingTable(newRoutingTable)
             .metaData(newMetaData);
         final RestoreInProgress restoreInProgress = allocation.custom(RestoreInProgress.TYPE);
+        ImmutableOpenMap.Builder<String, ClusterState.Custom> customsBuilder = null;
```

Comment:
move this line above the one with `final RestoreInProgress restoreInProgress = allocation.custom(RestoreInProgress.TYPE);`?